apps/web/lib/log-helper.ts

```diff
@@ -0,0 +1,12 @@
+import { redactPII } from "@/modules/ee/audit-logs/lib/utils";
```

Comment:
you just shifted the problem; for generic helpers we shouldn't import `ee` code as this requires an ee license to run (given by the license terms, not enforced in code)
If the redactPII function is a generic function not only used by `ee` code, we need to move this out of the ee module.

Reply:
Ah, right, I think we can use `redactPII` directly in `handleSsoCallback` since they’re both part of the `ee` code. We’re not getting much value from the `createDebugContext` function anyway. Does this make sense to you?